### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by rollerchloe**
Readme is updated, so that it can be used as a remote file in the documentation website.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
-[![CircleCI](https://circleci.com/gh/Open-Attestation/decentralized-renderer-react-template.svg?style=svg)](https://circleci.com/gh/Open-Attestation/decentralized-renderer-react-template)
-
 # React Template
 
-React boilerplate to write decentralize renderer.
+This is a React boilerplate to write the decentralize renderer.
```
Comment (yapyuyou) [inline README.md]:
```suggestion
This is a React boilerplate to create a decentralized renderer.
```

File: README.md [hunk 2]
```diff
@@ -39,82 +41,111 @@ $ npm run example:application # start embedded application
...
-- documents samples contains fake data to test the templates you build. You can create a typescript interface to make sure that your template uses the correct expected document
-  - create a type describing the kind of document you expect to render (for instance check `CustomTemplateCertificate` in `src/sample.tsx`)
-  - create a template consuming that interface, using the `TemplateProps` helper from `@govtechsg/decentralized-renderer-react-components (for instance check the template in`src/templates/customTemplate/customTemplate.tsx`)
-- shared components are located in `src/core` folder. For instance you can find a watermark example that will be displayed when printing the document
-- examples of how to add a watermark and qr code are located in `examples/template`
-- feel free to remove whatever you dont need
+- The `index.ts` file contains the logic to communicate with the host embedding your renderer. You might not need to update this file.
+- The template registry configuration is located in the `src/templates/index.tsx` file.
+- Templates are located in the `src/templates` folder. You can add as many template as required and structure the code accordingly.
```
Comment (yapyuyou) [inline README.md]:
```suggestion
- Templates are located in the `src/templates` folder. You can add as many templates as required and structure the code accordingly.
```

File: README.md [hunk 3]
```diff
@@ -39,82 +41,111 @@ $ npm run example:application # start embedded application
...
 
-- Remove `examples` folder completely
-- Remove line 5 with `../example` context from `.storybook/config.js` (so that storybook won't look for example files)
+- The uninstallation process will remove the `examples` folder completely.
+- It will remove line 5 with the `../example` context from `.storybook/config.js`, so that the storybook won't look for example files.
 
 ## License
 
-GPL-3.0
+The React template for the decentralized renderer is under the GPL-3.0 license.
```
Comment (yapyuyou) [inline README.md]: Yes can we update this to `Apache 2.0 license`?

**Review by yapyuyou — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rollerchloe) [reply README.md]: updated

File: README.md (hunk 2 above)
Comment (rollerchloe) [reply README.md]: updated to plural form

File: README.md (hunk 3 above)
Comment (rollerchloe) [reply README.md]: Yes

**Review by yapyuyou — APPROVED** [review]

**Pull request merged**